common/lib/xmodule/xmodule/templates/problem/checkboxes_response.yaml

```diff
@@ -3,21 +3,46 @@ metadata:
     display_name: Checkboxes
```

Comment:
I think what the .yaml does it provide the new starter template you get when you click the make-a-new-one button. As such, it's super high visibility and worthy of discussion. I could imagine it being less in-your-face about the extended hint features.